README.md

```diff
@@ -6,136 +6,54 @@
 
-Replace `<personal-access-token>` with the token you created in step 1. Alternatively you can omit `SUPABASE_ACCESS_TOKEN` in this config and instead set it globally on your machine. This allows you to keep your token out of version control if you plan on committing this configuration to a repository.
 
-The following options are available:
+### 2. Configure your MCP client
 
-- `--read-only`: Used to restrict the server to read-only queries and tools. Recommended by default. See [read-only mode](#read-only-mode).
-- `--project-ref`: Used to scope the server to a specific project. Recommended by default. If you omit this, the server will have access to all projects in your Supabase account. See [project scoped mode](#project-scoped-mode).
-- `--features`: Used to specify which tool groups to enable. See [feature groups](#feature-groups).
+The Supabase MCP server is hosted at `https://mcp.supabase.com/mcp` and supports the Streamable HTTP transport with OAuth authentication.
 
-If you are on Windows, you will need to [prefix the command](#windows). If your MCP client doesn't accept JSON, the direct CLI command is:
+The easiest way to connect your MCP client (such as Cursor) to your project is clicking [Connect](https://supabase.com/dashboard/project/_?showConnect=true&tab=mcp) in the Supabase dashboard and navigating to the MCP tab. There you can choose options such as [feature groups](#feature-groups), and generate one-click installers or config entries for popular clients.
 
-```shell
-npx -y @supabase/mcp-server-supabase@latest --read-only --project-ref=<project-ref>
-```
+![supabase-mcp-url-builder](https://github.com/user-attachments/assets/f77d4c40-24e2-49e2-aba8-2148b0b1788f)
 
-> Note: Do not run this command directly - this is meant to be executed by your MCP client in order to start the server. `npx` automatically downloads the latest version of the MCP server from `npm` and runs it in a single command.
-
-#### Windows
-
-On Windows, you will need to prefix the command with `cmd /c`:
+Most MCP clients store the configuration as JSON in the following format:
 
 ```json
 {
   "mcpServers": {
     "supabase": {
-      "command": "cmd",
-      "args": [
-        "/c",
-        "npx",
-        "-y",
-        "@supabase/mcp-server-supabase@latest",
-        "--read-only",
-        "--project-ref=<project-ref>"
-      ],
-      "env": {
-        "SUPABASE_ACCESS_TOKEN": "<personal-access-token>"
-      }
+      "type": "http",
+      "url": "https://mcp.supabase.com/mcp"
     }
   }
 }
 ```
 
-or with `wsl` if you are running Node.js inside WSL:
+Your MCP client will automatically prompt you to login to Supabase during setup. This will open a browser window where you can login to your Supabase account and grant access to the MCP client. Be sure to choose the organization that contains the project you wish to work with. In the future, we'll offer more fine grain control over these permissions.
```

Comment:
Copied this from the URL builder callout in https://github.com/supabase/supabase/pull/36867, can remove if redundant